draft-ietf-ppm-dap.md

```diff
@@ -822,12 +822,12 @@ to authenticate the aggregator they upload to.
 HTTPS provides confidentiality between the DAP client and the leader, but this
 is not sufficient since the helper's report shares are relayed through the
 leader. Confidentiality of report shares is achieved by encrypting each report
-share to a public key held by the respective aggregator. Clients fetch the
-public keys from each aggregator over HTTPS, allowing them to authenticate the
-server.
+share to a public key held by the respective aggregator using [HPKE].
+Clients fetch the public keys from each aggregator over HTTPS, allowing them to
+authenticate the server.
 
 Aggregators MAY require clients to authenticate when uploading reports. This is
-an effective mitigation against Sybil attacks in deployments where it is
+an effective mitigation against Sybil {{Dou2}} attacks in deployments where it is
```

Comment:
```suggestion
an effective mitigation against Sybil [Dou2] attacks in deployments where it is
```

Reply:
Don't we want {{Dou2}} so that the existence of the reference is enforced?